source/change-streams/change-streams.rst

```diff
@@ -178,13 +181,29 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
     /**
      * A document containing key:value pairs of names of the fields that were
      * changed, and the new value for those fields.
+     *
+     * Modifications to array elements are expressed via the dot notation (https://docs.mongodb.com/manual/core/document/#document-dot-notation).
+     * Example: an `update` which sets the element with index 0 in the array field named arrayField to 7 is reported as
+     *   "updatedFields": {"arrayField.0": 7}
      */
     updatedFields: Document;
 
     /**
      * An array of field names that were removed from the document.
      */
     removedFields: Array<String>;
+
+    /**
+     * Truncations of arrays may be reported using one of the following methods:
+     * either via this field or via the ‘updatedFields’ field. In the latter case the entire array is considered to be replaced.
+     *
+     * Example: an `update` which shrinks the array arrayField.0.nestedArrayField from size 8 to 5 may be reported as
+     *   "truncatedArrays": [{field: "arrayField.0.nestedArrayField", newSize: 5}]
+     *
+     * @note The method used to report a truncation is a server implementation detail.
+     * @since 4.7.0
+     */
+    truncatedArrays: Array<String>;
```

Comment:
Based on your example, should this be `Array<Document>` where each document is of the form `{field: <string>, newSize: <int>}`?

Reply:
I apologize, this is my mistake caused by copy-pasting. Yes, the type is `Array<Document`.